reference/api/search.md

```diff
@@ -735,3 +738,27 @@ Queries using `_geoPoint` will always include a `geoDistance` field containing t
 ```
 
 [You can read more about location-based sorting in our dedicated guide.](/learn/advanced/geosearch.md#sorting-results-with-geopoint)
+
+### Matching strategy
+
+**Parameter**: `matchingStrategy`
+**Expected value**: `last` or `all`
+**Default value**: `last`
+
+Defines the strategy used to match query terms in documents. `last` returns documents containing all the query words first. If there aren't enough documents to meet the requested `limit`, it will remove one query term at a time, starting from the end of the query term.
```

Comment:
```suggestion
Defines the strategy used to match query terms in documents. `last` returns documents containing all the query words first. If there aren't enough documents to meet the requested `limit`, it will remove one query term at a time, starting from the end of the query.
```